### PR TITLE
Update baselines for mdoc 5.8.9

### PR DESCRIPTION
**Pull request by gewarren**
mdoc 5.8.9 adds support for function pointers in C#.

### CODE REVIEW

File: xml/System.Web.DynamicData/DynamicFilter.xml [hunk 1]
```diff
@@ -417,9 +417,9 @@
         <Parameter Name="source" Type="System.Linq.IQueryable" Index="0" FrameworkAlternate="netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8" />
       </Parameters>
       <Docs>
-        <param name="source">The data source query.</param>
-        <summary>Returns the filter repeater.</summary>
-        <returns>The filter.</returns>
+        <param name="source">To be added.</param>
+        <summary>To be added.</summary>
+        <returns>To be added.</returns>
```
Comment (BillWagner) [inline xml/System.Web.DynamicData/DynamicFilter.xml:422]: @gewarren Changes like this do concern me. Any ideas why this happens?

**Review by BillWagner — COMMENTED** [review]
@gewarren 

The changes for the function pointer prototypes look good to me.

In other APIs, information looks like it was lost in this PR. Any ideas why that happened?

File: xml/System.Web.DynamicData/DynamicFilter.xml (hunk 1 above)
Comment (gewarren) [reply xml/System.Web.DynamicData/DynamicFilter.xml:422]: Good catch, this is a bug. I've logged https://dev.azure.com/ceapex/Engineering/_workitems/edit/598286.

**Review by BillWagner — APPROVED** [review]
Thanks again @gewarren 

This LGTM. Let's :shipit:

**Pull request merged**